posthog/batch_exports/sql.py

```diff
@@ -1,5 +1,7 @@
-CREATE_PERSONS_BATCH_EXPORT_VIEW = """
-CREATE OR REPLACE VIEW persons_batch_export AS (
+from django.conf import settings
+
+CREATE_PERSONS_BATCH_EXPORT_VIEW = f"""
+CREATE OR REPLACE VIEW persons_batch_export ON CLUSTER {settings.CLICKHOUSE_CLUSTER} AS (
```

Comment:
Noticed these were not created `ON CLUSTER`. The additional curly braces are for escaping.